src/lib/dialog/dialog-container.ts

```diff
@@ -45,6 +41,9 @@ export class MdDialogContainer extends BasePortalHost implements OnDestroy {
   /** Reference to the open dialog. */
   dialogRef: MdDialogRef<any>;
 
+  /** Whether the focus trap is active. */
+  focusTrapActive: boolean = false;
```

Comment:
Not sure what the style guide says about this, but wouldn't the `boolean` be inferred from the default value anyway @jelbourn?